src/bundle/Resources/views/fields/location/path.html.twig

```diff
@@ -0,0 +1,8 @@
+{% spaceless %}
+    {% for location in locations %}
+        <a href="{{ path('_ezpublishLocation', {'locationId': location.id}) }}">{{ ez_content_name(location.contentInfo) }}</a>
+        {% if not loop.last %}
+        /
```

Comment:
instead of generating the separator here, I think it should be defined in the CSS and the markup should be something like:

```html
<ul class="ez-breadcrumbs">
    <li class="ez-breadcrumbs-item"><a href="/whatever">Parent</li>
    <li class="ez-breadcrumbs-item"><a href="/whatever2">Element</li>
</ul>
```

normally the corresponding CSS is already there